upgrade/rendering-changes/4-6-0.md

```diff
@@ -0,0 +1,71 @@
+---
+title: 4.6.0
+description: Handle the rendering changes in the 4.6.0 release of the Telerik UI for Blazor components.
+page_title: Rendering Changes in the 4.6.0 Release
+slug: changes-in-4-6-0
+position: 1
+---
+
+# Rendering Changes in 4.6.0
+
+* [Icons](#icons)
+* [Form](#form)
+* [Carousel](#carousel)
+* [Filter](#filter)
+
+## Icons
+
+### Font Icons
+
+The font icons are detached from the Kendo UI Themes distribution. Read the [Font Icons do not render in Telerik UI for Blazor 4.6]({%slug icon-kb-font-icons-not-rendering%}) to see how to use Font icons in your application after the 4.6.0 Telerik UI for Blazor release. 
+
+### Changes in Icon Rendering
+
+The CSS classes applied to the built-in Font and SVG icons will be unified to include the `k-icon` CSS class.
+
+<table>
+    <thead>
+        <tr>
+            <th>UI for Blazor 4.5.0</th>
+            <th>UI for Blazor 4.6.0</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td style="vertical-align:top">
+<pre><code>
+&lt;span class="k-icon k-i-home"&gt;&lt;/span&gt;
+&lt;span class="k-svg-icon k-svg-i-home"&gt;&lt;/span&gt;
```

Comment:
Ideally, the before-after snippets should have the same number of lines. The SVG span doesn't follow this best practice.

Reply:
fixed